test/test_helper.rb

```diff
@@ -62,6 +62,12 @@ def unstub_project_callbacks
     Project.any_instance.unstub(:clone_repository)
     Project.any_instance.unstub(:clean_repository)
   end
+
+  def record_hooks(callback, &block)
+    called = []
+    Samson::Hooks.with_callback(callback, lambda{ |*args| called << args }, &block)
+    called
+  end
```

Comment:
How about adding a short comment above this method explaining its usage, or an example? It took me reading over it a few times and looking at an example to figure out its purpose.